### PR TITLE
Feature/#046 testperfil

### PR DESCRIPTION
**Pull request by migromarj**
Now to run the view tests from the console with the command 'python manage.py test' it is necessary to download the chrome driver (https://chromedriver.chromium.org/downloads) and add the folder in which it is located to the environment variables.

### CODE REVIEW

**Review by ezegonmac — APPROVED** [review]
All test run OK. I think there are no ones missing👍🏻

File: participant/tests.py [hunk 1]
```diff
@@ -1,3 +1,121 @@
-from django.test import TestCase
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
```
Comment (IsmaelP19) [inline participant/tests.py:6]: Lines 2, 5 and 6 should be deleted as they are unused imports.

**Review by IsmaelP19 — APPROVED** [review]
All tests running OK. I suggest you to define some string constants in order to reduce duplications of string literals.

Anyway, good job!! 🙌🚀

File: participant/tests.py (hunk 1 above)
Comment (migromarj) [reply participant/tests.py:6]: Thanks a lot! I hadn't realized that I had imports that I wasn't using, plus the 'Sonarlint' extension I use for vscode didn't tell me that they weren't used.

**Pull request merged**